Link to null safety migration article from evolution page as an example (#4048)
dart-lang/site-www: src/_guides/language/evolution.md

to null safety might take a while,
Dart uses language versioning to support
using non-null-safe code alongside null-safe code.
For an example of how an app or package can migrate to
a new language version

> with an incompatible feature—like was introduced with null safety—see 

Reviewer: ```suggestion
with an incompatible feature (null safety, for example), see 
```

I had to read this sentence multiple times because I thought it said "feature-like". It didn't *look* like an mdash until I went to edit the text.
